projects/wave-core/src/lib/operators/dialogs/data-repository/raster/source-dataset.component.html

```diff
@@ -28,7 +28,8 @@
             <ng-container matColumnDef="name">
                 <mat-header-cell *matHeaderCellDef>Channel</mat-header-cell>
                 <mat-cell *matCellDef="let channel">
-                    {{channel.name}}
+                    <span bind-innerHtml="channel.name | waveHighlightPipe:searchTerm | waveSafeHtml"></span>
+                    <!--{{channel.name}}-->
```

Comment:
Also here, you can delete the commented out code. We can recover it from version control if necessary.

Reply:
deleted